.github/workflows/cron.yml

```diff
@@ -52,7 +53,16 @@ jobs:
           npm install
           npm run snap
           git add ./data
-          git commit -m "Snapshot of data done!"
+          git commit -m "Snapshot of svm data done!"
+      - name: a0 snapshot
```

Comment:
Adding a new job is not a good idea because the snapshot on A0 might not execute if one of the earlier jobs fails, such as the SVM snapshot. They should be independent. Ideally, they should be executed in parallel